spras/meo.py

```diff
@@ -167,4 +179,7 @@ def parse_output(raw_pathway_file, standardized_pathway_file):
         # TODO what should be the edge rank?
         # Would need to load the paths output file to rank edges correctly
         df.insert(5, 'Rank', 1)  # Add a constant rank of 1
-        df.to_csv(standardized_pathway_file, columns=['Source', 'Target', 'Rank'], header=False, index=False, sep='\t')
+
+        df = readd_direction_col_mixed(df, "Type", "pd", "pp")
```

Comment:
Does it output mixed graphs or only directed graphs? The algorithm is designed to assign directions to undirected edges.

Reply:
We discussed this and the meaning of this part of the MEO readme
> The edges are annotated as pp for edges that were undirected in the original network and pd for edges that were originally directed.

MEO tracks the directionality of the original edges, but all of its output edges are directed. That makes its pathways quite different from all of the other SPRAS algorithms. However, to remain accurate to MEO's design we will also treat the output graph's as directed.